include/wx/graphics.h

```diff
@@ -1033,7 +1033,7 @@ class WXDLLIMPEXP_CORE wxGraphicsRenderer : public wxObject
                               const wxGraphicsMatrix& matrix = wxNullGraphicsMatrix) = 0;
 
     // sets the font
-    virtual wxGraphicsFont CreateFont( const wxFont &font , const wxColour &col = *wxBLACK ) = 0;
+    virtual wxGraphicsFont CreateFont( const wxFont &font , const wxColour &col = *wxBLACK, const wxSize &dpi = wxSize() ) = 0;
```

Comment:
Seeing that we operate with doubles everywhere in this code, maybe `dpi` argument should use `wxRealPoint` instead?